iet.so/src/iet_worker.cpp

```diff
@@ -228,12 +229,13 @@ bool IETWorker::do_iet_power_stress(void) {
         // check if stop signal was received
         if (rvs::lp::Stopping())
             break;
-        // get GPU's current average power
-        rsmi_status_t rmsi_stat = rsmi_dev_power_ave_get(smi_device_index , 0,
-                &last_avg_power);
 
+        cur_power_value = 0;
+
+        // get GPU's current average power
+        rsmi_status_t rmsi_stat = rsmi_dev_power_get(smi_device_index, &last_power, &type);
```

Comment:
do we need to check "type" value here?

Reply:
Not really because not much use in RVS as **type** will differ in MI300 vs Older GPUs. 
MI300 - RSMI_CURRENT_POWER - Power based on device
MI200 & others - RSMI_AVERAGE_POWER - Power based on GPU die

https://confluence.amd.com/display/BLTZ/MI-300+ROCm-SMI+Features